index.js

```diff
@@ -1,15 +1,12 @@
 /*jslint node: true */
 'use strict';
 
-var listify = function listify(list) {
+var listify = function listify(list, options) {
 	if (!Array.isArray(list)) {
 		throw new TypeError('requires an array');
 	}
+	options = options || {};
 
-	var options = arguments.length > 1 ? arguments[1] : null;
```

Comment:
This is intentional. The function's length is `1` because that's how many required arguments there are. I don't want `options` in the signature.